src/ui/CategoryFacet/CategoryValue.ts

```diff
@@ -80,14 +80,29 @@ export class CategoryValue implements CategoryValueParent {
     return this.categoryChildrenValueRenderer.children;
   }
 
+  public makeSelectable() {
```

Comment:
Hmm, this would depend on my commennt below, but would `makeAccessible` be a better if we are adding more than just the selectAction?

Reply:
Not 100% sure I understand your comment, but in this case it's not doing anything else than making it "selectable", either by mouse or keyboard.

So I think the name is fine ?